packages/web-components/src/select/select.ts

```diff
@@ -0,0 +1,206 @@
+import { attr, FASTElement, observable } from '@microsoft/fast-element';
+import { ARIAGlobalStatesAndProperties } from '@microsoft/fast-foundation';
+import { SelectSize } from './select.options.js';
+
+/**
+ * The base class used for constructing a fluent-slider custom element
+ * @public
+ */
+export class Select extends FASTElement {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleInput = this.handleInput.bind(this);
+    Object.assign(this, new ARIAGlobalStatesAndProperties()); // Assigns ARIA global states and properties to the element
+  }
+
+  // Attributes
+  /**
+   * The name of the select element
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: name
+   */
+  @attr
+  public name?: string;
+
+  /**
+   * The required boolean attribute of the select element
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: required
+   */
+  @attr({ mode: 'boolean' })
+  public required?: boolean;
+
+  /**
+   * The optional label of the select
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: label
+   */
+  @attr
+  public label?: string;
```

Comment:
Label as a string instead of a slot? I thought we discussed a slot to align with the other current implementations of form controls?

Reply:
You're right. This was a legacy bit that I missed removing.